### DISCUSSION TITLE
Hosting root website at custom domain AND account.github.io

### DISCUSSION THREAD

**Discussion by seen-network** [community/community#36143] [Other Feature Feedback, Questions, & Ideas]
Is it possible to configure Github Pages so your root site (ie. account.github.io NOT repo.github.io) is available at a custom domain AND account.github.io?

**Comment by yoannchaudet** [comment]
When you setup a custom domain for a repo, accessing the URL of the repo on the `github.io` domain will automatically redirect to the custom domain. There is no other way around.
